packages/tabs/src/Tabs.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// hixme-ui
+import Text from '@hixme-ui/text'
+import Container from '@hixme-ui/container'
+import theme from '@hixme-ui/theme'
+
+const {
+  backgroundColors,
+} = theme
+
+const defaultStyle = {
+  transition: 'all 0.25s ease',
+  borderBottom: '4px solid white',
+  borderBottomWidth: '0',
+  marginBottom: '4px',
+}
+
+const active = {
+  transition: 'all 0.25s ease',
+  borderBottom: `4px solid ${backgroundColors.blue}`,
+  borderBottomWidth: '4px',
+}
+
+class Tabs extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      tabIndex: 0,
+    }
+  }
+
+  componentDidMount() {
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({
+      tabIndex: this.props.initWithIndex,
+    })
+  }
+
+  handleTabClick = (clickedIndex) => {
+    this.setState({
+      tabIndex: clickedIndex,
+    })
+  }
+
+  render() {
+    return (
+      <Container flex noPadding direction='column' width='100%' height='100%'>
+        <Container flex noPadding>
+          {this.props.list && this.props.list.map((tab, index) => {
+            const classState = (this.state.tabIndex === index)
+            ? active
+            : defaultStyle
+            return (
+              <Container
+                flex
+                noPadding
+                key={`${tab.name}${this.props.id}tab`}
+                margin='0 20px 0 0'
+              >
+                <div style={classState}>
```

Comment:
Just a suggestion of an alternative way of doing this, but I imagine you could make this div a styled component, pass in the tabIndex and index, and do the different styles in there.